extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -473,16 +483,6 @@ export class SchemaCompareDialog {
 			return uniqueValues;
 		}, []);
 
-		// reverse list so that most recent connections show first
-		values.reverse();
-
-		// move server of current connection to the top of the list so it is the default
-		if (idx >= 1) {
-			let tmp = values[0];
-			values[0] = values[idx];
-			values[idx] = tmp;
-		}
-
 		return values;
 	}
 
```

Comment:
So now the order is - reverse(), then find current index and bring it to front (default) then reduce. Right?

Reply:
yes